### ISSUE TITLE
external monitor crashes niri

### ISSUE THREAD

**Issue by frederic-romagne** [YaLTeR/niri#242]
Niri is crashing, (reloading to GDM), after connecting an external monitor on my laptop. For information, it works with 2 other monitors. I tried with and without display settings in the config file to no avail.

journalctl --user -xeu niri.service:
```
niri[20379]: 2024-03-04T13:20:53.987554Z DEBUG niri::niri: putting output eDP-1 at x=-1920 y=750
niri[20379]: 2024-03-04T13:20:53.987563Z DEBUG niri::backend::tty: connecting connector: HDMI-A-1
niri[20379]: 2024-03-04T13:20:53.987569Z DEBUG niri::backend::tty: picking mode: Mode { name: "1920x1200", clock: 154000, size: (1920, 1200), hsync: (1968, 2000, 2080), vsync: (1203, 12>
niri[20379]: thread 'main' panicked at /builddir/build/BUILD/niri-0.1.2/vendor/edid-rs/src/lib.rs:641:13:
niri[20379]: attempt to subtract with overflow
niri[20379]: stack backtrace:
niri[20379]: note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
systemd[19545]: niri.service: Main process exited, code=exited, status=101/n/a
```

### System Information

* niri version: niri 0.1.2 (62892d6) (Archlinux)

* GPU: Intel Alderlake P

* CPU: Intel i7-1260P

Cheers

**Comment by YaLTeR** [comment]
Hey, I just fixed it yesterday: 2750b2038bed5495bcdfeacc7be25267d15ceab1. Same issue as #239. Funny how it hasn't come up and then it came up for two people in a row

**Comment by frederic-romagne** [comment]
Sorry, I looked for a similar bug only few days ago, I wouldn't have post a duplicate...
Keep up the good work, Niri nicely fits my workflow, great job!
